rasa/shared/core/flows/flow.py

```diff
@@ -399,6 +403,44 @@ def step_from_json(flow_step_config: Dict[Text, Any]) -> FlowStep:
         return BranchFlowStep.from_json(flow_step_config)
 
 
+def build_flow_steps_list(steps: List[Dict[Text, Any]]) -> List[FlowStep]:
+    flow_steps: List[FlowStep] = []
+
+    for i, step_config in enumerate(steps):
+        if step_config.get("next") is None:
+
+            if i == len(steps) - 1:
+                step_config["next"] = END_STEP
+            else:
+                step_config["next"] = steps[i + 1].get("id")
```

Comment:
we should raise here if that is `None` - this shouldn't happen. do we validate the steps before they get passed to this method?